.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: Build
+
+on: push
+
+jobs:
+  setup-linkcheck-build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    - name: Checkout sub modules
+      run: |
+        git submodule init && git submodule update
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get update
+        sudo bash setup.sh --quiet-mode
+
+    - name: add-git-credentials
+      run: |
+        git config --global user.name "circle-ci"
+        git config --global user.email "circle-ci@nomail.com"
+
+    - name: Build
+      run: |
+        make linkcheck
+        make html
+      #
+      # uses: maraskar/sphinx-action@master
+      # with:
+      #   # build-command: make linkcheck
+      #   pre-build-command: make linkcheck
+      #   build-command: make html
+      #   docs-folder: "./"
```

Comment:
This should be removed?

Reply:
Yes, has been updated in the current iteration.